pkg/apis/operator/v1alpha1/types_garden.go

```diff
@@ -166,6 +170,42 @@ type Volume struct {
 	MinimumSize *resource.Quantity `json:"minimumSize,omitempty"`
 }
 
+// CertManagement configures the cert-management component for issuing TLS certificates
+// from an ACME server.
+type CertManagement struct {
+	// Deployment contains configuration for deploying the cert-controller-manager.
+	// +optional
+	Deployment *CertManagementDeployment `json:"deployment,omitempty"`
```

Comment:
How about `Config` instead of `Deployment`?

Reply:
renamed